.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This should be ignored in .gitignore

Reply:
ignored

Reply:
Please delete this file from Pull Request.

Reply:
deleted. Please check.